components/remote_transmitter.rst

```diff
@@ -48,6 +48,26 @@ Configuration variables:
 -  **id** (*Optional*, :ref:`config-id`): Manually specify
    the ID used for code generation. Use this if you have multiple remote transmitters.
 
+Automations:
+------------
+
+- **on_transmit** (*Optional*, :ref:`Automation <automation>`): An automation to perform before
+  data is sent. Useful if the radio / IR hardware needs to change state or power on.
+- **on_complete** (*Optional*, :ref:`Automation <automation>`): An automation to perform after
+  data has been sent. Useful if the radio / IR hardware needs to change state or power off.
+
+.. code-block:: yaml
+
+    # Example automation
+    remote_transmitter:
+      ...
+      on_transmit:
+        then:
+          - lambda: 'id(radio_id)->start_tx();'
+      on_complete:
+        then:
+          - lambda: 'id(radio_id)->stop_tx();'
+
```

Comment:
We should not have an example of something that doesnt yet exist

Reply:
I was hoping that [PR ](https://github.com/esphome/esphome/pull/7483) would get merged, however I can rework the docs.

Reply:
Yeah good point but not sure what else to show as an example. Could just show a print I guess?

Reply:
Ok I will show a switch or gpio. Updating now